homeassistant/components/qwikswitch/binary_sensor.py

```diff
@@ -13,8 +13,8 @@
 
 async def async_setup_platform(hass, _, add_entities, discovery_info=None):
     """Add binary sensor from the main Qwikswitch component."""
-    if discovery_info is None:
-        return
+    # will not be called if no info
```

Comment:
See Paulus made a similar comment, all these are still required

Reply:
ok. that makes sense. not an expert on these

any idea how to make a test case that will go through these flows? would it be with a config passed to hass as {'light':['platform': 'qwikswitch']}

Reply:
when unit tests don't cover it, i had in the past simple cases where i put retrun instead of return and all the tests still pass, so i am pretty big on the concept of 100% coverage

Reply:
Static type checking (Pylint and flake8) helps with this, but you can call setup_component with the config you suggested

Reply:
ok. restored, and test added